pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
```

Comment:
Maybe we should also move entirely to peotry. I've never used it though. Do you happen to have any experience with it @neuneck ?

Reply:
I don't. If it's just a tool that doesn't conflict with other workflows, we can give it a shot. 

Myself, I'm a fond user of virtualenvwrapper. For me, using more complex tools than pip has always lead to some issue earlier or later (tried pipenv and conda), so I'm weary of buying into python project management tools that aren't easily compatible with the core lib tools.

Reply:
Agreed. Was just curious if you had any experience with it.